docs/health/index.rst

```diff
@@ -2,62 +2,137 @@
 
 pyATS Health Check
 ==================
-Collect and monitor the state of your devices as your testscript is executing with pyATS Health Check. It can collect traceback, core files, etc. pyATS Health Check is all yaml driven and it is based on :ref:`Blitz<blitz>`. All Blitz functionalities are fully supported in health check.
+By using pyATS Health Check, it is possible to check the CPU load, memory usage, detection of specific log messages, and whether or not the core file created in the event of a crash or malfunction is generated.
 
-The health check is driven by a `health_file` which is provided at run time. There is two different mechanism to run the checks:
+pyATS Health Check currently supports the following 4 checks by default.
 
-1. Part of a pre/post processor of a section or testcase.
-2. Part of another process continuously (Coming up in September).
+.. list-table::
+   :header-rows: 1
 
-pyATS Health Check leverages existing :ref:`Blitz<blitz>` style YAML format and use same format and capabilities what Blitz has.
-The actions in pyATS Health Check yaml, those actions will be added to pyATS job's each section as pre/post processors. Pre, Post or both processors can be specified in the yaml.
+   * - health check
+     - description
+   * - cpu
+     - cpu load check. check total of cpu load with threshold 90% by default.
+   * - memory
+     - memory usage check. check total of memory usage with threshold 90% by default.
+   * - logging
+     - keyword check in show logging output. default keywords are traceback, Traceback, TRACEBACK
+   * - core
+     - check if core file is generated on device. just check by default. Please use `--health-remote-device` to copy the core file to remote server.
 
-.. note::
 
-     See the detail of :ref:`Blitz<blitz>`
+Using pyATS Health Check is very easy. Just list the above check names with `--health-checks` in the current pyats command.
 
 .. code-block:: bash
 
-    # Quick start Guide!
-    # ------------------
-    # 1. Create the health yaml
-    # 2. Add the arguments
-    # 3. (Optional) Run it and hope no issue is found by the pyATS Health Check!
+    pyats run job <job file> --testbed-file /path/to/testbed.yaml --health-checks cpu memory logging core
+
+If you want to check only cpu, please specify just only `cpu` to `--health-checks`. **Also, importantly, the pyATS Health Check by `--health-checks` runs as a post-processor after each test case.**
```

Comment:
the " the pyATS Health Check by" seems to be missing some words..

Reply:
this is the sentence `the pyATS Health Check by `--health-checks` runs as a post-processor after each test case.` please correct if any issue...